packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -119,6 +119,13 @@ Media ID (or media IDs if multiple is true) to be selected by default when openi
 - Type: `Number|Array`
 - Required: No
 
+### append
```

Comment:
We can now remove this prop documentation :)

Reply:
😅